tests/acceptance/test_cases/acceptance_test_case.py

```diff
@@ -0,0 +1,55 @@
+"""
+Base class for acceptance tests of the API. Acceptance tests run on the
+entire server stack, and check that TopChef works end-to-end
+"""
+import unittest
+from topchef import APP_FACTORY
```

Comment:
Nevermind, it's used in ``unit`` testing, as it should be! @MichalKononenko doesn't know what he's talking about!